_includes/logo.md

```diff
@@ -4,11 +4,15 @@
         {%- assign platform = "dotnet" -%}
     {%- when "kotlin" -%}
         {%- assign platform = "android" -%}
+    {%- when "typescript" -%}
+        {%- assign platform = "js" -%}
 {%- endcase -%}
 {%- assign file_name = platform | prepend: '/' | append: '.svg' -%}
 {%- for file in site.static_files -%}
     {%- if file.path contains '/assets/img/logos/' and file.path contains file_name -%}
 ![{{ include.platform }}](/assets/img/logos/{{ platform }}.svg)
+    {%- else -%}
+<!-- {{ include.platform }} -->
```

Comment:
Is this just adding a `else` statement to print out a HTML comment?

Reply:
Yes, it's just to know what `include.platform` is so it's easier to add an icon if we don't have any.